pages/api/lnurlp/[username]/index.js

```diff
@@ -13,8 +14,9 @@ export default async ({ query: { username } }, res) => {
     minSendable: 1000, // Min amount LN SERVICE is willing to receive, can not be less than 1 or more than `maxSendable`
     maxSendable: 1000000000,
     metadata: lnurlPayMetadataString(username), // Metadata json which must be presented as raw string here, this is required to pass signature verification at a later step
+    commentAllowed: LNURLP_COMMENT_MAX_LENGTH, // LUD-12 Comments for payRequests https://github.com/lnurl/luds/blob/luds/12.md
     tag: 'payRequest', // Type of LNURL
-    nostrPubkey: getPublicKey(process.env.NOSTR_PRIVATE_KEY),
-    allowsNostr: true
+    nostrPubkey: process.env.NOSTR_PRIVATE_KEY ? getPublicKey(process.env.NOSTR_PRIVATE_KEY) : undefined,
+    allowsNostr: !!process.env.NOSTR_PRIVATE_KEY
```

Comment:
I made these changes only because I didn't have `NOSTR_PRIVATE_KEY` setup locally. Shouldn't cause an issue in prod, but without these changes, I couldn't do this work. Up to you if you want to keep them, or force devs to configure a valid nostr private key.